### PR TITLE
F.A.Q

### PR DESCRIPTION
**Pull request by tpluscode**

### CODE REVIEW

File: _includes/faq.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+## FAQ
+
+### Why is there no agenda?
+
+We call this "Everybody talks everybody listens"™️.
+Each participant is given a slot to present the subject of their choosing. 
+Depending on the attendance and session the slot may be between 5 and 30 minutes long.
```
Comment (switzersc) [inline _includes/faq.md]: I would update to "session topic" to clarify that the topic impacts how long the slot will be.

Comment (tpluscode) [reply _includes/faq.md]: @switzersc mhm. By "session" I meant something else. Group of presentations. A block perhaps?

File: _includes/faq.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+## FAQ
+
+### Why is there no agenda?
+
+We call this "Everybody talks everybody listens"™️.
+Each participant is given a slot to present the subject of their choosing.
```
Comment (switzersc) [inline _includes/faq.md:6]: I'd add a bit more detail after this - something like: "There are no restrictions on the subject of the talk: participants can talk about the biggest challenge they're facing, a cool tool they've used or built, or lessons learned from working on a new team -- or none of the above. Sometimes a participant simply poses a question to the entire group, and lively conversation ensues."

Comment (switzersc) [reply _includes/faq.md:6]: Oh I see you talk about that below. Take it or leave it then!

**Review by switzersc — APPROVED** [review]
+1

File: _includes/faq.md (hunk 1 above)
Comment (switzersc) [reply _includes/faq.md]: Ohh I see. I think you're right - the best word probably is session.

Comment (BigBlueHat) [reply _includes/faq.md]: Hrm... we've used "session talks" for the 30 minute length ones in the past. How about...
"Depending on the attendance we'll have speaking slots between 5 and 30 minutes long."

File: _includes/faq.md (hunk 2 above)
Comment (tpluscode) [reply _includes/faq.md:6]: I do take it! Incorporated into that other answer

File: _includes/faq.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+### Do I have to talk about RESTful APIs?
+
+Certainly not. It's perfectly fine to present REST-ish, REST-like and even GraphQL ;). 
+In fact, you can even talk about something else if you think it will be interesting to 
+other participants (see the question below). 
+
+### Do I have to present an API?
+
+Not necessarily. You can talk about what you are passionate about. Surely, API-related
+subjects are preferred but there have been exceptions.
```
Comment (BigBlueHat) [inline _includes/faq.md]: The last two questions are pretty similar. Can we combine them? Folks are certainly welcome to talk about 🐱's, etc. Our objective is to build better community within the REST and API space--hence, everyone talks; everyone listens. 😃

**Review by BigBlueHat — APPROVED** [review]
Wee tweak request, but can happen later (if ever).

**Pull request merged**